lib/griffin/interceptors/server/newrelic_interceptor.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+gem 'newrelic_rpm'
+
+module Griffin
+  module Interceptors
+    module Server
+      class NewRelicInterceptor < GRPC::ServerInterceptor
+        def initialize(ignored_services: [])
+          @ignored_services = ignored_services.map(&:service_name)
+        end
+
+        Request = Struct.new(:path, :user_agent, :request_method)
+
+        def request_response(method: nil, call: nil, **)
+          # return yield if NewRelic::Agent.instance.started?
```

Comment:
intented comment?

Reply:
It's a mistake. Thansk! Fixed it https://github.com/cookpad/griffin-interceptors/pull/2/files#diff-f438d4b8f1fecb68a9d4022a6b8614f1R16